ogc/features/json.go

```diff
@@ -130,36 +131,59 @@ func (jf *jsonFeatures) createFeatureCollectionLinks(selfFormat string, collecti
 			Href:  featuresURL.toSelfURL(collectionID, engine.FormatJSON),
 		})
 	}
+
 	links = append(links, domain.Link{
 		Rel:   "alternate",
 		Title: "This document as HTML",
 		Type:  engine.MediaTypeHTML,
 		Href:  featuresURL.toSelfURL(collectionID, engine.FormatHTML),
 	})
+
 	if cursor.HasNext {
-		links = append(links, domain.Link{
-			Rel:   "next",
-			Title: "Next page",
-			Type:  engine.MediaTypeGeoJSON,
-			Href:  featuresURL.toPrevNextURL(collectionID, cursor.Next, engine.FormatJSON),
-		})
+		switch currentFormat {
```

Comment:
evt, om iets minder dubbele code te hebben, de links var maken en alleen de Href aanpassen in de switch

Reply:
het is href en type die aangepast moeten worden, heb het even bekeken maar het scheelt niet echt aan code.